docs/chains/darwinia-smart-chain.md

```diff
@@ -8,7 +8,7 @@ title: Darwinia Smart Chain
 Darwinia Smart Chain is a substrate based ethereum compatible smart contract platform where contract developers can easily migrate
 smart contracts from the Ethereum chain to run their business. In this article we will cover a number of topics that are important to contract developers, such as network configuration, pre-compiles and so on. In addition, we have set up a free test network, the pangoro network, where the latest features will be fully tested before they are added to the Darwinia Smart Chain. The user can obtain test tokens at [Pangoro Test Token Faucet](https://apps.darwinia.network/?network=pangoro).
```

Comment:
`precompiles` maybe better.

`precompiles` or `precompiled contracts` is a convention name in the evm world.

Reply:
done